server/src/resolvers/getVariantsResolver/utils/liftOver.ts

```diff
@@ -0,0 +1,93 @@
+import { AssemblyId, VariantQueryDataResult } from '../../../types';
+import resolveAssembly from '../utils/resolveAssembly';
+const fs = require('fs/promises');
+const tmpdir = require('os').tmpdir;
+const path = require('path');
+const promiseExec = require('util').promisify(require('child_process').exec);
+
+const createTmpFile = async () => {
+  const filename = Math.random().toString(36).slice(2);
```

Comment:
You don't need this. `mkdtemp` is already putting this file in a unique directory.